spec/fixtures/proofing/lexis_nexis/ddp/error_response.json

```diff
@@ -2,5 +2,6 @@
   "error_detail": "service_type",
   "request_id":"1234-abcd",
   "request_result":"fail_invalid_parameter",
-  "review_status":"REVIEW_STATUS"
+  "review_status":"REVIEW_STATUS",
+  "tmx_summary_reason_code": ["Identity_Negative_History"]
```

Comment:
did we get this from checking production logs?

Reply:
Yes, I checked the production logs and also from LN we have a lot of others as well but I just mock this only.

Reply:
This could be a constant in DocAuthHelper, SAMPLE_TMX_SUMMARY_REASON_CODE or something like that. Along with GOOD_SSN, etc. Since it's used in 3 tests (so far). You might still have to have the explicit string in the fixtures.